src/main/java/com/hillel/items_exchange/config/SecurityConfig.java

```diff
@@ -1,10 +1,14 @@
 package com.hillel.items_exchange.config;
 
+import com.hillel.items_exchange.security.jwt.JwtAuthenticationEntryPoint;
 import com.hillel.items_exchange.security.jwt.JwtConfigurator;
 import com.hillel.items_exchange.security.jwt.JwtTokenProvider;
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
I don't see this annotation into the code. Would you mind to clear unused imports (for Intellij Idea: Ctrl + Alt + O)?

Reply:
clean code in commit "e56e52f"